dockstore-webservice/src/main/java/io/dockstore/webservice/core/Token.java

```diff
@@ -87,6 +90,11 @@ public long getId() {
     }
 
     @JsonProperty
+    public String getToken() {
```

Comment:
Just add a comment to both (getToken and getContent) saying that the duplication is for backwards compatibility